assembl/static2/js/app/actions/phaseActions.js

```diff
@@ -1,18 +1,20 @@
-export const updateCurrentPhaseIdentifier = (currentIdentifier) => {
+// @flow
+
+export const updateCurrentPhaseIdentifier = (currentIdentifier: string) => {
```

Comment:
it would be nice to also have typing for return values here. Look at this commit for example 609e7c for an example. You can then use these in reducers (lots of examples in `resourcesCenter` actions and reducers.

Reply:
woops :(

Reply:
I chose to put action types in a separate file but I think it should be a folder if it grows (but it is not a good time to do this)